### DISCUSSION TITLE
Impossible to create the root directory on s3 bucket

### DISCUSSION THREAD

**Discussion by afrasiyabhaider** [spatie/image#168] [Q&A]
Same code is working for local storage driver but its not working for S3 @freekmurze 

<img width="448" alt="image" src="https://user-images.githubusercontent.com/25869253/181664765-77ec3fcc-86d1-4307-af7d-605cd600faf9.png">
<img width="537" alt="image" src="https://user-images.githubusercontent.com/25869253/181664805-33731662-e672-4623-b69b-fd3712e00db9.png">

**Comment by DoubleClickDesignLtd** [comment]
Did you find a solution to this issue? I'm having similiar issues
